cabal-install/src/Distribution/Client/CmdTest.hs

```diff
@@ -235,7 +226,7 @@ data TestProblem
   | -- | The 'TargetSelector' refers to a component that is not a test-suite
     TargetProblemComponentNotTest PackageId ComponentName
   | -- | Asking to test an individual file or module is not supported
-    TargetProblemIsSubComponent PackageId ComponentName SubComponentTarget
+    TargetProblemIsSubComponent PackageId ComponentName
```

Comment:
Can't you get rid of this constructor as well?

Reply:
Good catch!